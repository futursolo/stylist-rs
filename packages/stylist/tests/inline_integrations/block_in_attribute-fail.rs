fn main() {
    let is_an_expression = "black";
    let _ = stylist::css! {
        .outer {
            border: ${is_an_expression};
            background-color: {should_be_expression};
            color: {another_expression};
        }
    };
}
