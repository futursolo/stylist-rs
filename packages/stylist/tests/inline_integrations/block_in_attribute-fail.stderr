error: expected a valid part of an attribute, got a block. Did you mean to write `${..}` to interpolate an expression?
 --> $DIR/block_in_attribute-fail.rs:6:31
  |
6 |             background-color: {should_be_expression};
  |                               ^^^^^^^^^^^^^^^^^^^^^^

error: expected a valid part of an attribute, got a block. Did you mean to write `${..}` to interpolate an expression?
 --> $DIR/block_in_attribute-fail.rs:7:20
  |
7 |             color: {another_expression};
  |                    ^^^^^^^^^^^^^^^^^^^^
